nav2_system_tests/models/turtlebot3_waffle_depth_camera/turtlebot3_waffle_realsense/model.sdf

```diff
@@ -0,0 +1,528 @@
+<?xml version="1.0" ?>
```

Comment:
1- Enabling the depth sensor and the camera sensor can slow down the system. 
2- The basic system test doesn't need any camera.

I am working on this PR. I will get rid of all the mesh files and use simple shapes for the robot and the sensors as @SteveMacenski and @crdelsey suggested.

Reply:
Or you can just download the assets instead of redoing the URDF, I dont think Carl or I care which and it might be faster. We just dont want pure duplication

Reply:
There was a xacro release for dashing recently, so you might be able to conditionally add sensors and gazebo plugins with an include like this:

`<xacro:include filename="$(find nav2_system_tests)/models/turtlebot3_waffle_depth_camera/turtlebot3_waffle_realsense.gazebo.xacro"/>`

Reply:
We don't want to make turtlebot code a dependency of the navigation2 stack.

Thought for system tests I think that's fine, you need _something_ to be the example.